packages/fluentui/docs/src/examples/components/Toolbar/Visual/ToolbarExampleVariables.shorthand.tsx

```diff
@@ -1,10 +1,18 @@
-import { Toolbar } from '@fluentui/react-northstar';
+import { Toolbar, toolbarMenuBehavior, Accessibility, ToolbarMenuBehaviorProps } from '@fluentui/react-northstar';
 import * as React from 'react';
 import { CallVideoIcon, MoreIcon, SkypeLogoIcon } from '@fluentui/react-icons-northstar';
 
+// behavior is overridden, as focus was causing inconsistency in displaying focus outline for screener tests
+const notAutoFocusToolbarMenuBehavior: Accessibility<ToolbarMenuBehaviorProps> = props => {
```

Comment:
Let's remove changes not related to this PR

Reply:
screener test was unstable and after agreement with @layershifter and @miroslavstastny we did the workaround the focus is not going to the menu when is mounted :)